docs/rule/kotlin_library.soy

```diff
@@ -32,7 +32,7 @@ of the compiled class files and resources.
 
 {call buck.arg}
   {param name: 'srcs' /}
-  {param default: '[]' /}
+  {param default: '{}' /}
```

Comment:
Was the intentional? Seems that you want this as part of your `kapt_apoptions`

Reply:
oof, good catch. yeah, i meant it for the kapt_apoptions param. thanks!